test/e2e/common/setup-shell.sh

```diff
@@ -25,7 +25,7 @@ function wait-for-cmd() (
     while ! (eval "$cmd") ; do
         if (($wait_counter > 10)); then
             echo "Failed to ${cmd}."
-            kubectl -n wds1-system logs -l control-plane=controller-manager
+            echo $(kubectl -n wds1-system logs -l control-plane=controller-manager)
```

Comment:
Why wrap the `kubectl` command into an `echo` command?

Reply:
I was expecting a log, so thought it's not working for some reason. Which wasn't the issue.